gpii/node_modules/contextManager/src/ContextManagerUtilities.js

```diff
@@ -0,0 +1,101 @@
+/*!
+GPII Context Manager Utilities
+
+Copyright 2014 Raising the floor - international
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/gpii/universal/LICENSE.txt
+*/
+
+/* global require */
+"use strict";
+
+var fluid = fluid || require("infusion"),
+    gpii = fluid.registerNamespace("gpii");
+
+fluid.registerNamespace("gpii.contextManager.utils");
+fluid.registerNamespace("gpii.contextManager.transforms");
+
+gpii.contextManager.utils.transformTypeLookup = {
+    "http://registry.gpii.net/conditions/inRange": "gpii.contextManager.transforms.inRange",
+    "http://registry.gpii.net/conditions/timeInRange": "gpii.contextManager.transforms.timeInRange"
+
+};
+
+// Transforms required for condition evaluation:
+fluid.defaults("gpii.contextManager.transforms.inRange", {
```

Comment:
These transforms should be moved into Transformer.js and its namespace so that we have them all in one place when we come to evaluate them for lifting up into the framework